network/p2pNetwork.go

```diff
@@ -269,7 +289,16 @@ func NewP2PNetwork(log logging.Logger, cfg config.Local, datadir string, phonebo
 	}
 	log.Infof("P2P host created: peer ID %s addrs %s", h.ID(), h.Addrs())
 
-	net.service, err = p2p.MakeService(net.ctx, log, cfg, h, la, net.wsStreamHandler, pubsubMetricsTracer{})
+	// TODO: remove after consensus v41 takes effect.
```

Comment:
Should we consider a consensus flag here to explicitly remove support?

Reply:
no, just wait until the entire network gets upgraded, it is similar to how we removed v1, v2.0 from wsnet over time.